Using different artifact repositories in a restricted environment (#1160)
codenvy/che-docs: src/main/pages/che-7/end-user-guide/proc_configuring-python-to-use-a-non-standard-registry.adoc

// Module included in the following assemblies:
//
// using-python-artifact-repositories

[id="configuring-python-to-use-a-non-standard-registry_{context}"]
= Configuring Python to use a non-standard registry

> You can specify a non-standard repository for the Python pip tool to use by setting the `PIP_INDEX_URL` environment variable.

Reviewer: ```suggestion
To specify a non-standard repository for use by the Python pip tool, set the `PIP_INDEX_URL` environment variable.
```

.Procedure

> * In your devfile, configure the `PIP_INDEX_URL` environment variable for both the language support and the dev container components:

Reviewer: "dev container" is 'developer', 'development', or a term/name?
